concepticondata/conceptlists/Carvalho-2021-206.tsv

```diff
@@ -0,0 +1,207 @@
+Carvalho-2021-206-51	51	person	683	PERSON
+Carvalho-2021-206-52	52	spirit/soul	811	SOUL
+Carvalho-2021-206-53	53	man	1554	MAN
+Carvalho-2021-206-54	54	woman	962	WOMAN
+Carvalho-2021-206-55	55	father	1217	FATHER
+Carvalho-2021-206-56	56	father2	1217	FATHER
+Carvalho-2021-206-57	57	mother	1216	MOTHER
+Carvalho-2021-206-58	58	mother2	1216	MOTHER
+Carvalho-2021-206-59	59	son	1620	SON
+Carvalho-2021-206-60	60	daughter	1357	DAUGHTER
+Carvalho-2021-206-61	61	grandson	1618	GRANDSON
+Carvalho-2021-206-62	62	granddaughter	1619	GRANDDAUGHTER
+Carvalho-2021-206-63	63	father-in-law	1055	FATHER-IN-LAW
+Carvalho-2021-206-64	64	mother-in-law	1050	MOTHER-IN-LAW
+Carvalho-2021-206-65	65	nephew/niece	3890	NEPHEW OR NIECE
+Carvalho-2021-206-66	66	younger brother	1760	YOUNGER BROTHER
+Carvalho-2021-206-67	67	younger sister	1761	YOUNGER SISTER
+Carvalho-2021-206-68	68	grandmother	1496	GRANDMOTHER
+Carvalho-2021-206-69	69	grandfather	1383	GRANDFATHER
+Carvalho-2021-206-70	70	pet	3780	PET
+Carvalho-2021-206-71	71	paca (rodent sp.)	316	PACA
+Carvalho-2021-206-72	72	capybara	314	CAPYBARA
+Carvalho-2021-206-73	73	rat	1490	RAT
+Carvalho-2021-206-74	74	agouti	313	AGOUTI
+Carvalho-2021-206-75	75	coati	2389	COATI (COATIMUNDI)
+Carvalho-2021-206-76	76	peccary (sp.)1	2390	PECCARY
+Carvalho-2021-206-77	77	peccary (sp.)2	2390	PECCARY
+Carvalho-2021-206-78	78	anteater	181	ANTEATER
+Carvalho-2021-206-79	79	tapir	312	TAPIR
+Carvalho-2021-206-80	80	Capuchin monkey		
+Carvalho-2021-206-81	81	howler monkey>	433	HOWLER MONKEY
+Carvalho-2021-206-82	82	sloth	2446	SLOTH
+Carvalho-2021-206-83	83	snake	730	SNAKE
+Carvalho-2021-206-84	84	boa		
+Carvalho-2021-206-85	85	alligator	1581	ALLIGATOR
+Carvalho-2021-206-86	86	turtle	112	TURTLE
+Carvalho-2021-206-87	87	ant	587	ANT
+Carvalho-2021-206-88	88	isula (ant sp.)		
+Carvalho-2021-206-89	89	termite	883	TERMITE
+Carvalho-2021-206-90	90	wasp	1517	WASP
+Carvalho-2021-206-91	91	mosquito	1509	MOSQUITO
+Carvalho-2021-206-92	92	bee/honey		
+Carvalho-2021-206-93	93	fly	1504	FLY (INSECT)
+Carvalho-2021-206-94	94	butterfly	1791	BUTTERFLY
+Carvalho-2021-206-95	95	cockroach	1575	COCKROACH
+Carvalho-2021-206-96	96	spider	843	SPIDER
+Carvalho-2021-206-97	97	scorpion	1538	SCORPION
+Carvalho-2021-206-98	98	tick	1527	TICK
+Carvalho-2021-206-99	99	louse	1392	LOUSE
+Carvalho-2021-206-100	100	worm1	1219	WORM
+Carvalho-2021-206-101	101	worm2	1219	WORM
+Carvalho-2021-206-102	102	fish (generic)	227	FISH
+Carvalho-2021-206-103	103	piraña (fish sp.)	317	PIRANHA
+Carvalho-2021-206-104	104	crab	2348	CRAB
+Carvalho-2021-206-105	105	bat	1793	BAT
+Carvalho-2021-206-106	106	bird (sp.)1	937	BIRD
+Carvalho-2021-206-107	107	bird (sp.)2	937	BIRD
+Carvalho-2021-206-108	108	bird (sp.)3	937	BIRD
+Carvalho-2021-206-109	109	hummingbird	1589	HUMMINGBIRD
+Carvalho-2021-206-110	110	duck	1360	DUCK
+Carvalho-2021-206-111	111	eagle	1905	EAGLE
+Carvalho-2021-206-112	112	owl	735	OWL
+Carvalho-2021-206-113	113	tree	906	TREE
+Carvalho-2021-206-114	114	palm (sp.)	1181	PALM TREE
```

Comment:
Is this about a specific type of palm? PALM refers to any species from 2 families, wheres "sp." sounds like it refers to one specific species. Same likely with the 3 birds.
As it's a cognate set-based list, we can probably expect that the terms they mapped to these concepts do not necessarily even refer to the same species in all the languages.

Reply:
As with some other glosses here, (sp.) appears to mean 'species' in order to disambiguate the concept in this case from 'palm of the hand'. Since there are no other palm trees in the list, I mapped it to the general and most obvious PALM TREE.

Reply:
Hm, but what would be the disambiguating factor for the other instances of sp.? Peccary, bird, paca, isula and piraña at least are unambiguous in that they refer to animals.

Reply:
I think that this could also be mentioned in the list's description.

Reply:
> Peccary (sp.)1’ and ‘peccary (sp.)2’ refer to two species of the genus, according to Brandão
and Facundes (2007:115).

It looks like some glosses from the target languages that were taken from dectionaries refer to different species of the same (generic) animal. The numbers next to 'bird' and 'peccary' then are presumably an attempt to generalize over them for comparison reasons.

Reply:
The question now is whether the duplicate concepts marked with (sp.) such as 'bird' and 'peccary' should be mapped just once. Keep in mind that the glosses were grouped not by the species they refer to but by the cognates they share.